receiver/vcenterreceiver/client.go

```diff
@@ -103,6 +106,24 @@ func (vc *vcenterClient) Datacenters(ctx context.Context) ([]mo.Datacenter, erro
 	return datacenters, nil
 }
 
+func (vc *vcenterClient) getVsphereAPIVersion() string {
+	return vc.vimDriver.ServiceContent.About.ApiVersion
+}
+
+func (vc *vcenterClient) vsphereAPIVersionMeetsMin(minVsphereAPIVersion string) bool {
```

Comment:
@BominRahmani I like this better here! Should be made public and then could use a few additional unit tests (< > = versions). 😄  Sorry to keep this going. After that should be good though!